server-worker/pom.xml

```diff
@@ -101,6 +101,96 @@
             <artifactId>spark-sql_${scala.binary.version}</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-client</artifactId>
+            <version>${hadoop.version}</version>
```

Comment:
Maybe we can delete version and exclusions? As parent pom has already defined them.

Reply:
> Maybe we can delete version and exclusions? As parent pom has already defined them.

Have remove this part